assets/js/components/legacy-notifications/DashboardCoreSiteAlerts.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * DashboardCoreSiteAlerts component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+const { useSelect } = Data;
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import Notification from './notification';
+import {
+	acceptNotification,
+	dismissNotification,
+} from './site';
+
+const MAX_SECONDS_FOR_SURVEY = 5;
+
+const DashboardCoreSiteAlerts = () => {
+	const [ ready, setReady ] = useState( false );
+	const [ hasSurveys, setHasSurveys ] = useState( false );
+	const [ startTime, setStartTime ] = useState( false );
+	const surveys = useSelect( ( select ) => select( CORE_USER ).getCurrentSurvey() );
+	const notifications = useSelect( ( select ) => select( CORE_SITE ).getNotifications() );
+
+	useEffect(
+		() => {
+			setStartTime( Date.now() );
```

Comment:
```suggestion
			setStartTime( Date.now() );
```

Could this be set as a state variable or a `useRef` on initial render rather than in a `useEffect`? Curious why this needs to be here/updated after `hasSurveys` changes.

Reply:
Sure, updated in https://github.com/google/site-kit-wp/pull/3603/commits/7a55d6a9850349c158d8fb1b6bb66e9e38aab13d